create_and_lock_environment.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+os_name=$(uname)
+if [[ ! $os_name == *"Linux"* ]]; then
+    echo "ERROR: cannot run AML environment locking in non-linux environment. Windows users can do this using WSL - https://docs.microsoft.com/en-us/windows/wsl/install"
```

Comment:
There's something off here - even with WSL, I get uname being "Linux".

Reply:
That's the correct behaviour as WSL uses a Linux kernel?